### PR TITLE
cas 350 - Update proposal routes

### PR DESCRIPTION
**Pull request by germanurrus**
- Update route configuration for proposal url

### CODE REVIEW

File: frontend/packages/client/src/pages/index.js [hunk 1]
```diff
@@ -41,9 +44,6 @@ export default function AppPages() {
             <Route exact path="/proposal/create">
               <ProposalCreate />
             </Route>
-            <Route path="/proposal/:proposalId">
-              <Proposal />
-            </Route>
```
Comment (germanurrus) [inline frontend/packages/client/src/pages/index.js:46]: - should we leave this one and redirect to the new created route?

File: frontend/packages/client/src/pages/index.js [hunk 2]
```diff
@@ -32,6 +32,9 @@ export default function AppPages() {
             <Route exact path="/community/create">
               <CommunityCreate />
             </Route>
+            <Route path="/community/:communityId/proposal/:proposalId">
+              <Proposal />
+            </Route>
             <Route path="/community/:communityId/edit" exact={true}>
```
Comment (germanurrus) [inline frontend/packages/client/src/pages/index.js:38]: we have three routes starting with the same path, I'm holding to create a separate folder with the nested route config structure, maybe if this grows we should do that

**Comment by dbslone** [comment]
@germanurrus I think we should also move the `/proposal/create` endpoint to be `/community/:id/proposal/create`

**Comment by germanurrus** [comment]
> @germanurrus I think we should also move the `/proposal/create` endpoint to be `/community/:id/proposal/create`

@dbslone updates where done to change create proposal url

**Review by dbslone — APPROVED** [review]

**Pull request merged**